docs/orchestration/concepts/kv_store.md

```diff
@@ -0,0 +1,123 @@
+# KV Store
+
+Key Value Store is your managed metadata database within Prefect Cloud.
+
+**Keys** are strings. **Values** are json blobs.
+
+The number of key value pairs allowed is limited by license, starting with 50 pairs on the Free tier. Values are limited to 1 MB in size.
+
+Key value pairs can be configured via the Prefect CLI, Python client, API, and UI.
+
```

Comment:
Could we add a small example of a simple flow using kv? (Or would that maybe be better in a blog post?)